.ruby-version

```diff
@@ -1 +1 @@
-3.2.0
+3.2.2
```

Comment:
I think 3.2.0 was picked to match production, I would leave that as-is

Reply:
We do have 3.2.2 installed, so it should be safe to use here.